includes/helpers/functions/general.php

```diff
@@ -991,6 +995,28 @@ function wpbdp_get_fee_plan( $plan_id ) {
     return WPBDP__Fee_Plan::get_instance( $plan_id );
 }
 
+/**
+ * Check if there are enabled premium fee plans.
+ * This does a count for all enabled fee plans with a price greater than 0.
+ *
+ * @since x.x
+ *
+ * @return bool
+ */
+function wpbdp_has_premium_fees() {
```

Comment:
Is there a place we could put this inside of a class?

Reply:
Moved this to `WPBDP_Fees_API` class, now called as `WPBDP_Fees_API::has_premium_fees()`